frontend/src/container/TraceDetail/utils.ts

```diff
@@ -98,3 +98,25 @@ export const getTreeLevelsCount = (tree: ITraceTree): number => {
 
 	return levels;
 };
+
+export const formGetTraceItemUrlParams = (
+	paramsMap: Map<string, string | null>,
+): string => {
+	let urlParams = '';
+	let index = 0;
+	paramsMap.forEach((value, key) => {
+		if (index === 0) {
+			if (value) {
+				urlParams = `?${key}=${value}`;
+			} else {
+				urlParams = `?${key}=`;
```

Comment:
handle with try and catch as there is `URI malformed`

for reference: `k8s_pod_name%20NIN%20(%27asd%27)asd3daafc+q32easdc%%%`